### PR TITLE
Fix issue with CONDA_ENV and using powerline

### PR DESCRIPTION
**Pull request by cstich**
This fixes issue [3982](https://github.com/conda/conda/issues/3982). Powerline detects the conda environment on it's own but doesn't actually put the string into PS1. I am not sure what it exactly does behind the scenes, but the outcome is that you get the conda enviornment twice in your shell prompt. FYI, I have just tested this with zsh.

### CODE REVIEW

File: shell/activate.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+/#!/usr/bin/env bash
```
Comment (kalefranz) [inline shell/activate.sh]: Was this intentional?

Comment (cstich) [inline shell/activate.sh]: No I don't think so

Comment (cstich) [inline shell/activate.sh]: Should be fixed

**Comment by kalefranz** [comment]
target is 4.3.x via #4925

**Pull request closed without merging**